OPERATING.md

```diff
@@ -0,0 +1,114 @@
+# Operating
+
+How to operate the GP2GP adaptor
+
+## Adaptor Process
+
+TODO: Sequence diagram "user journey" of Spine messages, tasks, and GPCC requests
+
+## Logging and Tracing
+
+```
+yyyy-MM-dd HH:mm:ss.SSS Level=INFO Logger=u.n.a.g.e.m.AgentDirectoryExtractor ConversationId=18262a7b-cb4c-4aef-8dd8-7ffc3dda67aa TaskId=ad4e53cc-9b4e-4e07-8937-f0f91887b21d Thread=java-thread-1 Message="The log message"
+```
+
+* Level: The logging level DEBUG/INFO/WARN/ERROR
+* Logger: The name of the Java class that emitted the message
+* ConversationId: The identifier correlating all GP2GP message for a single patient transfer
+* TaskId: The unique identifier of the task
+* Message: The log message
+
+## Object Storage (AWS S3 / Azure Blob)
+
+TODO: Document storage scheme
+
+## AMQP Message Broker Requirements
+
+* The broker must be configured with a limited number of retries and deadletter queues
```

Comment:
In **Using Azure Service Bus** we write it as "dead-letter". Could be nice to be consistent.